programming/networking/distributed/distributed-smooth-node.rst

```diff
@@ -10,3 +10,39 @@ the client.
 
 This DO can be used if the repository has been initialized with the direct.dc
 added to the dc files list.
+
+The make use of the smoothing functionality and automatically broadcasting of
```

Comment:
"*To* make use of", I assume this should be.

Reply:
Also, "automatic broadcasting", I think, since "automatic" is an adjective to the noun "broadcasting" here.